File: Assignment Programs/Assignment37/program3.java
import java.util.*;

class Pattern
{
	public void Pattern(int iRow, int iCol)
	{
		int i = 0, j = 0;
		for(i = 1;i <= iRow;i++)
		{
			for(j = iCol;j >= 1;j--)
			{
				System.out.print(j+"\t");
			}
			System.out.println();
		}
	}
	
}

class program3
{
	public static void main(String args[])
	{
		int iNo1 = 0, iNo2 = 0;
		Scanner sobj = new Scanner(System.in);
		
		System.out.println("Enter number of rows");
		iNo1 = sobj.nextInt();
		
		System.out.println("Enter number of columns");
		iNo2 = sobj.nextInt();
		
		Pattern pobj = new Pattern();
		pobj.Pattern(iNo1, iNo2);
	}
}

File: Assignment Programs/Assignment15/program3.c
#include<stdio.h>
void Pattern(int iRow, int iCol)
{
	int i = 0, j = 0;
	if(iRow != iCol)
	{
		return;
	}
	for(i = 1;i<=iRow;i++)
	{
		for(j = 1;j<=iCol;j++)
		{
			if((i+j) == (iRow+1))
			{
				printf("*\t");
			}
			else if((j==1) || (j == iRow) || (i==1) || (i==iRow))
			{
				printf("*\t");
			}
			else
			{
				printf(" \t");
			}
		}
		printf("\n");
	}
}
int main()
{
	int iValue1 = 0, iValue2 = 0;
	printf("Enter number of rows and columns\n");
	scanf("%d %d",&iValue1, &iValue2);
	Pattern(iValue1, iValue2);
	return 0;
}